src/host/_stream.h

```diff
@@ -40,7 +40,7 @@ Return Value:
                                             const BOOL fKeepCursorVisible,
                                             _Inout_opt_ PSHORT psScrollY);
 
-#define LOCAL_BUFFER_SIZE 100
+constexpr auto LOCAL_BUFFER_SIZE = 100;

```

Comment:
This is used only in `_stream.cpp`. I have no idea why it's in the header file.

Reply:
I just checked the history. The reason is historical artifact behind `_stream.h` (there wasn't a _stream.cpp in the past and it was a weird double-compiled thing for Western v. Eastern languages). I'm OK with this.